tools/api-markdown-documenter/src/test/snapshots/html/simple-suite-test/default-config/test-suite-a/testemptyinterface-interface.html

```diff
@@ -1,30 +1,19 @@
-<!DOCTYPE html>
+<!doctype html>
 <html lang="en">
   <head>
-    <meta charset="utf-8" />
+    <meta charset="utf-8">
   </head>
   <body>
     <section>
-      <h1>
-        TestEmptyInterface
-      </h1>
+      <h1>TestEmptyInterface</h1>
       <section>
-        <p>
-          <a href='./'>Packages</a> &gt; <a href='./test-suite-a'>test-suite-a</a> &gt; <a href='./test-suite-a/testemptyinterface-interface'>TestEmptyInterface</a>
-        </p>
+        <p><a href="./">Packages</a> > <a href="./test-suite-a">test-suite-a</a> > <a href="./test-suite-a/testemptyinterface-interface">TestEmptyInterface</a></p>
```

Comment:
Here a `&gt;` changed to a `>` which feels like it'd break something when rendering this HTML doc.

Reply:
I believe the spaces around `>` make it safe in HTML. I'll confirm this, though. I believe only `<` characters need to be unilaterally escaped, and `>` is more context dependent. But I'll double check this.

Reply:
Confirmed.

![image](https://github.com/user-attachments/assets/797a86f8-ca4d-46a0-b872-4889baff16e1)

Reply:
The confirmation is on the brackets for the generic type? The opening one is `&lt;` and the closing one `>`?

If so, good to know it works... I think I'd still prefer them both escaped for consistency if possible, but won't block on it.

Reply:
That's correct. And yeah, I agree with you there. Unfortunately, `hast-util-to-html` doesn't let you configure this, and they seem to have taken the policy of minimal escaping. Overall, I prefer leveraging the external tool over maintaining our own rendering. And it's a widely used library, so we can have some confidence that they're on top of issues.